modules/installation-osp-fixing-subnet.adoc

```diff
@@ -24,22 +24,39 @@ The IP range that the installation program uses by default might not match the N
 +
 [source,terminal]
 ----
-$ python -c 'import yaml
+$ python -c 'import os
+import sys
+import yaml
+import re
+re_os_net_id = re.compile(r"{{\s*os_net_id\s*}}")
+os_net_id = os.getenv("OS_NET_ID")
+path = "common.yaml"
+facts = None
+for _dict in yaml.safe_load(open(path))[0]["tasks"]:
+    if "os_network" in _dict.get("set_fact", {}):
+        facts = _dict["set_fact"]
+        break
+if not facts:
+    print("Cannot find `os_network` in common.yaml file. Make sure OpenStack resource names are defined in one of the tasks.")
+    sys.exit(1)
+os_network = re_os_net_id.sub(os_net_id, facts["os_network"])
+os_subnet = re_os_net_id.sub(os_net_id, facts["os_subnet"])
 path = "install-config.yaml"
 data = yaml.safe_load(open(path))
 inventory = yaml.safe_load(open("inventory.yaml"))["all"]["hosts"]["localhost"]
 machine_net = [{"cidr": inventory["os_subnet_range"]}]
 api_vips = [inventory["os_apiVIP"]]
 ingress_vips = [inventory["os_ingressVIP"]]
-ctrl_plane_port = {"network": {"name": inventory["os_network"]}, "fixedIPs": [{"subnet": {"name": inventory["os_subnet"]}}]}
-if inventory.get("os_subnet6"): <1>
+ctrl_plane_port = {"network": {"name": os_network}, "fixedIPs": [{"subnet": {"name": os_subnet}}]}
```

Comment:
Might be out of scope but does this config also apply to the IPv6/IPv4 dual-stack networking option, where IPv6 is the primary network and IPv4 the secondary network?

Reply:
I believe it's out of scope, but feel free to chime in, @gryf.